src/components/elements/table.js

```diff
@@ -0,0 +1,47 @@
+import factory from './factory'
+import cx from 'classnames'
+
+export default ({ api, property }) => {
+  return (
+    <div className='ct-table-container'>
+        <label className='ct-element-label ct-table-label'>{property.title}</label>
+        <table className='ct-table'>
+        { property.columns &&
+        <thead>
+            <tr className='ct-table-header-row'>
+          { property.columns.map((column, rowIndex) => {
+            return (
+              <th key={column.id} className='ct-table-header'>
+                {column.title}
+              </th>
+              )
+          })
+          }
+            </tr>
+          </thead> }
+          <tbody>
+          {property.rows.map((row, rowIndex) => {
+            return (
+              <tr key={rowIndex} className='ct-table-row'>
+                  {row.map(cell => {
+                    const Element = factory(cell)
+                    return (
+                      <td key={cell.id} className='ct-table-cell'>
```

Comment:
I know @hfjallemark mentioned this but this component would be nice to split up into smaller components like Row, Cell etc.